components/constraints.js

```diff
@@ -12,8 +12,11 @@ class Constraints extends Component {
     if (!constraints) return <div />;
     return (
       <ul style={{ listStyle: 'none', paddingLeft: 0 }}>
-        {(constraints.get('default') || constraints.get('default') === 0) &&
-          <li>default value: {constraints.get('default')}</li>
+        {constraints.has('default') &&
+          (constraints.get('type') === 'boolean' ?
+            (constraints.get('default') ? 'default value: true' : 'default value: false') :
```

Comment:
```
default value: {constraints.get('default')}
```

We had partly this before and that was the reason of this bug. Because JavaScript/Immutable.js doesn't translate `false` into string and prints nothing.